tests/behat/features/capabilities/event/filter-community-events.feature

```diff
@@ -0,0 +1,50 @@
+@api @javascript @event @stability @stability-1 @filter-community-events
+Feature: Filter community events
+  Benefit: Ability to filter events on All events page
+  Role: As a Verified
+  Goal/desire: I want to filter events that have all day checkbox activated
+
+  Scenario: Successfully use filters for ongoing and upcoming events with today's end date
+    Given users:
+      | name               | mail             | status | timezone | roles    |
+      | regular_user       | some@example.com | 1      | UTC      | verified |
+    Given events with non-anonymous author:
+      | title                     | body        | field_content_visibility | field_event_date    | field_event_date_end | field_event_all_day	 | langcode |
+      | Upcoming or Ongoing Event | lorem ipsum | public                   | today               | today                | 1                    | en       |
+    When I am logged in as "regular_user"
+    Given I am on the event overview
+    And I should see "Filter" in the "Sidebar second"
+    And I should see "Ongoing and upcoming events"
+    And I click radio button "Ongoing and upcoming events"
+    And I press "Filter"
+    And I should see "Upcoming or Ongoing Event" in the "Main content"
```

Comment:
Finally we use `Then` to signal our assertion, again leaving a blank line before it for readability.

```suggestion

    Then I should see "Upcoming or Ongoing Event" in the "Main content"
```

As an aside, I usually try to avoid `in the "Region"` because it couples the test to how our theme is built more than I like. In this case it's needed because the name of the event is similar to the name of the radio button, you can get around that by naming your event something different like "My awesome Llama party", so that the string is unique on the page.

Reply:
Fixed